config/environments/development.rb

```diff
@@ -73,4 +73,10 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
+  config.action_mailer.delivery_method = :letter_opener
+  
+  config.action_mailer.perform_deliveries = true
```

Comment:
here we don't need newlines because these are settings for action mailer means they fall in same category so you can write these without extra new lines.